packages/salesforcedx-vscode-core/src/conflict/conflictNode.ts

```diff
@@ -49,8 +58,32 @@ export class ConflictNode extends vscode.TreeItem {
 
 export class ConflictFileNode extends ConflictNode {
   constructor(conflict: ConflictFile, parent: ConflictNode) {
-    super(conflict.fileName, vscode.TreeItemCollapsibleState.None, parent);
+    let collapsibleState: vscode.TreeItemCollapsibleState;
+    if (conflict.remoteLastModifiedDate || conflict.localLastModifiedDate) {
+      collapsibleState = vscode.TreeItemCollapsibleState.Expanded;
+    } else {
+      collapsibleState = vscode.TreeItemCollapsibleState.None;
+    }
+    super(conflict.fileName, collapsibleState, parent);
     this._conflict = conflict;
+    this.iconPath = new vscode.ThemeIcon('go-to-file');
```

Comment:
For this icon I actually recommend `ThemeIcon.File` - this uses the file icon and utilizes whatever file icon theme users have set.

Reply:
I see! updated. ;D